### ISSUE TITLE
Atmosphere Support

### ISSUE THREAD

**Issue by GiovanH** [jakibaki/sys-netcheat#25]
Hekate is not planning to update to 6.2. Consequently, sys-netcheat's reliance on hekate-specific loading is a problem. 

Atmosphere can, of course, load kip sysmodules. However, when loading sys-netcheat, the console hangs at the switch logo, assumedly due to lack of atmosphere support.

sys-netcheat should be updated to allow atmosphere loading.

**Comment by buitim** [comment]
Hekate was just updated for 6.2. However, Switch freezes on boot with sys-netcheat and latest Atmosphere

**Comment by jfmherokiller** [comment]
> 
> 
> Hekate was just updated for 6.2. However, Switch freezes on boot with sys-netcheat and latest Atmosphere

Same issue here just tested it today freeze on switch logo.

**Comment by rfortunatti** [comment]
Same, it freezes on Switch logo when trying to boot with the sys-netcheat.kip.

Also tried to boot using Hekate, modyfing the hekate_ini and pointing to the fusee_primary. Now Atmosphere 0.8.2 boots through, but i have no response from netcheat on the network.

**Comment by lborruto** [comment]
Same for me, using Kosmos 11.5 w/ Atmosphere 0.8.2 + Hekate 4.5.

**Comment by buitim** [comment]
Sorry to ping you @jakibaki, but if it helps to debug at all, ReiNX works with sys-netcheat. However, I think it uses an older commit from Atmosphere before the sleep fix was implemented. Using ReiNX, I was able to boot and modify values but could not get out of sleep mode. Essentially #22 

EDIT: [This commit](https://github.com/Reisyukaku/ReiNX/commit/032e5c244b072ce81cf271cceea2676284f6ddb8) works. Was just able to successfully boot with sys-netcheat and modify values in Let's Go Pikachu. Of course sleep is broken in this commit.
